csm/core/controllers/information.py

```diff
@@ -0,0 +1,74 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import json
+from marshmallow import Schema, fields, validates_schema
+from marshmallow.exceptions import ValidationError
+from csm.core.controllers.view import CsmView, CsmResponse
+from cortx.utils.log import Log
+from csm.common.errors import InvalidRequest
+from csm.core.blogic import const
+from csm.common.errors import CsmNotFoundError
+from csm.core.controllers.validators import ValidationErrorFormatter
+
+class VersionValidationSchema(Schema):
+    # Define schema here
+    requires = fields.List(fields.Str, data_key=const.REQUIRES, required=True)
+
+    @validates_schema
+    def invalidate_empty_values(self, data, **kwargs):
+        """Invalidate the empty strings."""
+        for key, value in data.items():
+            if value is not None and not str(value).strip():
+                raise ValidationError(f"Empty value for {key}")
+
+
+@CsmView._app_routes.view("/api/v2/version/compatibility/{resource}/{resource_id}")
+class VersionInformationView(CsmView):
+    """
+    Version compatiblity validation for REST API implementation.
+
+    POST: Validate version compatibilty
+    """
+    def __init__(self, request):
+        super().__init__(request)
+        self._service = self.request.app[const.INFORMATION_SERVICE]
+
+    async def post(self):
+        """POST REST implementation for Validating version compatibility."""
+        Log.debug("Handling POST request for Version compatibility Validation.")
```

Comment:
Why debug? f you want to log in debug please include some data points. Otherwise log it in info

Reply:
changing it to info